### PR TITLE
Add GH actions for alertmanager integration

### PR DESCRIPTION
**Pull request by aizuddin85**
### What does this PR do?
Adding GH actions for build and push to redhat-cop registry.

### How should this be tested?
As per GH CI.

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/tool-integrations

### CODE REVIEW

File: .github/workflows/ci-cd-am-tele.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    - uses: docker/build-push-action@v1
+      with:
+        path: alertmanager-notifier-gateway/alertmanager-telegram-gateway/
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+        repository: redhat-cop/alertmanager-telegram-gateway
```
Comment (oybed) [inline .github/workflows/ci-cd-am-tele.yml:27]: This doesn't seem to be an existing quay repo. Please submit issue to request one before we merge this PR:
https://github.com/redhat-cop/org/issues/new/choose

File: .github/workflows/ci-cd-am-line.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    - uses: docker/build-push-action@v1
+      with:
+        path: alertmanager-notifier-gateway/alertmanager-line-gateway/
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+        repository: redhat-cop/alertmanager-line-gateway
```
Comment (oybed) [inline .github/workflows/ci-cd-am-line.yml:27]: This doesn't seem to be an existing quay repo. Please submit issue to request one before we merge this PR:
https://github.com/redhat-cop/org/issues/new/choose

**Review by oybed — CHANGES_REQUESTED** [review]
Please see inline

File: .github/workflows/ci-cd-am-tele.yml (hunk 1 above)
Comment (aizuddin85) [reply .github/workflows/ci-cd-am-tele.yml:27]: https://github.com/redhat-cop/org/issues/199 submitted.

File: .github/workflows/ci-cd-am-line.yml (hunk 2 above)
Comment (aizuddin85) [reply .github/workflows/ci-cd-am-line.yml:27]: https://github.com/redhat-cop/org/issues/199 submitted.

**Pull request closed without merging**